torchmultimodal/models/mdetr/model.py

```diff
@@ -257,3 +260,236 @@ def mdetr_resnet101(
         extra_query_embeddings,
     )
     return mdetr
+
+
+class MultiHead(nn.Module):
+    def __init__(self, heads: nn.ModuleDict):
+        super().__init__()
+        self.heads = heads
+
+    def forward(
+        self,
+        embeddings: Tensor,
+    ) -> Dict[str, Tensor]:
+        if embeddings.size(0) != len(self.heads):
```

Comment:
just curious, why do u need a nn module for this

Reply:
It gets removed in the next PR 🙂